utils/utils.go

```diff
@@ -1044,3 +1046,27 @@ func ForkVersionAtEpoch(epoch uint64) *types.ForkVersion {
 		PreviousVersion: MustParseHex(Config.Chain.Config.GenesisForkVersion),
 	}
 }
+
+func LogError(infoIdentifier string, err error) *logrus.Entry {
+	var fileIdentifier, lineIdentifier, functionIdentifier string
+
+	pc, fullFilePath, line, ok := runtime.Caller(1)
```

Comment:
There should be an option to use Caller(2) as well, I need this feature in case of an uncaught panic, where Caller(1) will be panic.go, which is not a good info at all.

Reply:
You now have to set the skip parameter when calling the new log functions.